src/Apps/Search/FilterState.tsx

```diff
@@ -70,6 +70,10 @@ const isDefaultFilter = (filter, value): boolean => {
     return value === "-decayed_merch"
   }
 
+  if (filter === "medium") {
```

Comment:
@kierangillen could you do me a favor and verify that the browser history & URLs do what you'd expect them to when you select/deselect a medium filter?

Reply:
Made a slight update and it's good now, you can see it in the new gif in the description.